app/controllers/payments/OutstandingPaymentsController.scala

```diff
@@ -60,24 +59,24 @@ class OutstandingPaymentsController @Inject() (
     val outstandingCharges = financialData.outstandingCharges
 
     outstandingCharges
```

Comment:
Since you are refactoring things, the `val outstandingCharges = financialData.outstandingCharges` assignment is not really needed. We can directly use `financialData.outstandingCharges` here.
```suggestion
    financialData.outstandingCharges
```